common/test/acceptance/tests/studio/test_studio_textbooks.py

```diff
@@ -28,6 +31,7 @@ def setUp(self, is_staff=True):
 
         self.textbook_view_page = TextbookViewPage(self.browser, self.course_id)
 
+    @flaky  # TNL-5772
```

Comment:
Yes, @estute and I addressed it in https://github.com/edx/edx-platform/pull/13756